### PR TITLE
Authorize onboarding

### PR DESCRIPTION
**Pull request by balloob**
Tons of changes:

 - Add new mixin to provide translations without a `hass` object
 - Translations for onboarding and authorize pages
 - Authorize will now have always an auth provider open, always selects the first one automatically
 - Some code splitting
 - Move some static UI into the HTML files

screenshots-login

### CODE REVIEW

File: src/auth/ha-auth-flow.js [hunk 1]
```diff
@@ -2,46 +2,64 @@ import { PolymerElement } from '@polymer/polymer/polymer-element.js';
 import '@polymer/paper-button/paper-button.js';
 import { html } from '@polymer/polymer/lib/utils/html-tag.js';
 import '../components/ha-form.js';
-import EventsMixin from '../mixins/events-mixin.js';
+import localizeLiteMixin from '../mixins/localize-lite-mixin.js';
```
Comment (awarecan) [inline src/auth/ha-auth-flow.js]: Uppercase?

File: src/auth/ha-pick-auth-provider.js [hunk 2]
```diff
@@ -4,11 +4,12 @@ import { html } from '@polymer/polymer/lib/utils/html-tag.js';
 import { PolymerElement } from '@polymer/polymer/polymer-element.js';
 
 import EventsMixin from '../mixins/events-mixin.js';
+import localizeLiteMixin from '../mixins/localize-lite-mixin.js';
```
Comment (awarecan) [inline src/auth/ha-pick-auth-provider.js]: Uppercase?

**Review by awarecan — COMMENTED** [review]
Shall MFA translation move to front-end as well

**Comment by balloob** [comment]
It should remain in backend if we want to depend on custom MFA modules

For authorize page I decided to not support it now, because it is a leak if the auth page can see which things are loaded.

**Pull request merged**